### PR TITLE
Add Carthage support

### PR DESCRIPTION
**Pull request by Jachobsen**
Thank you for providing this great resource!

I've been using my fork with Carthage for a few months now but don't see any reason why it shouldn't be a part of your repo, so everyone can choose to use Carthage.

It references the current swift file, so there shouldn't be any maintenance overhead.

See more at [Supporting Carthage for your framework](https://github.com/Carthage/Carthage#supporting-carthage-for-your-framework).

### CODE REVIEW

File: NibDesignable/NibDesignable/Info.plist [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	<key>CFBundleIdentifier</key>
+	<string>com.justabeech.$(PRODUCT_NAME:rfc1034identifier)</string>
+	<key>CFBundleInfoDictionaryVersion</key>
+	<string>6.0</string>
+	<key>CFBundleName</key>
+	<string>$(PRODUCT_NAME)</string>
+	<key>CFBundlePackageType</key>
+	<string>FMWK</string>
+	<key>CFBundleShortVersionString</key>
+	<string>1.0</string>
```
Comment (mbogh) [inline NibDesignable/NibDesignable/Info.plist]: Does this influence the Carthage version or is it also tag based?

File: NibDesignable/NibDesignableTests/NibDesignableTests.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import UIKit
```
Comment (mbogh) [inline NibDesignable/NibDesignableTests/NibDesignableTests.swift]: This file should either be deleted and the test target as well or the tests from the demo project should be move here.

**Comment by mbogh** [comment]
Thank you for the effort, I will just try out the carthage support.

README.md should also be updated with Carthage instructions :) I can do this if you prefer.

File: NibDesignable/NibDesignable/Info.plist (hunk 1 above)
Comment (Jachobsen) [inline NibDesignable/NibDesignable/Info.plist]: It's solely tag/SHA based.

**Comment by Jachobsen** [comment]
Done :)

**Pull request merged**